app/internal/emotion.py

```diff
@@ -0,0 +1,91 @@
+import text2emotion as te
+from typing import Dict, Tuple, NamedTuple, Union
+
+from app.config import (
+    CONTENT_WEIGHTS,
+    LEVEL_OF_SIGNIFICANCE,
+    TITLE_WEIGHTS)
+
+
+EMOTIONS = {"Happy": "&#128515",
+            "Sad": "&#128577",
+            "Angry": "&#128544",
+            "Fear": "&#128561",
+            "Surprise": "&#128558"}
+
+Emoticon = NamedTuple("Emoticon", [("dominant", str), ("score", float),
+                      ("code", str)])
+
+
+def get_weights(emotion: str, title_emotion: Dict[str, float],
+                content_emotion: Dict[str, float] = None) -> float:
+    if not content_emotion:
+        return title_emotion[emotion]
+    return (title_emotion[emotion] * TITLE_WEIGHTS +
+            content_emotion[emotion] * CONTENT_WEIGHTS)
+
+
+def score_comp(emotion_score: float, dominant_emotion: Emoticon, emotion: str,
+               code: str, flag: int) -> Tuple[Emoticon, int]:
+    """
+    score comparison between emotions.
+    returns the dominant and if equals we flag it
+    """
```

Comment:
What if `emotion_score < dominant_emotion.score`? 

`flag` is not defined

Reply:
when `emotion_score < dominant_emotion.score`  the input flag will be returned